### ISSUE TITLE
መዝሙረ፡ ዳዊት፡ ምሰለ፡ መዝሙረ፡ ክርስቶስ፡

### ISSUE THREAD

**Issue by Ralph-Lee-UK** [BetaMasaheft/Documentation#1701]
With a psalter that contains the psalms of Christ, the overall work would be simple be Psalter LIT2701Dawit and then the sub items, the first would be LIT2005Mazmur which includes both the Psalms of David and the Psalms of Christ?

**Comment by DenisNosnitsin1970** [comment]
Sorry, are you sure about Mazmura Krestos? Maybe it is Mazmura Dengel?

**Comment by DenisNosnitsin1970** [comment]
Irrespective of this, I think that Psalter united with another text, esp if this combined text has a certain circulation, 
represent a special version of the Psalter (or maybe of the Psalms of David, more exactly speaking), and should get an independent ID. So it is not the combination of the two unmarked works. I understand https://betamasaheft.eu/Guidelines/?q=versions&id=definitionWorks in this way. I did not remember now if we have already such Psalter versions, we might have them, a little hidden maybe, please check the DB.

**Comment by thea-m** [comment]
Yes, I agree with @DenisNosnitsin1970 . The few example manuscripts which we already have encoded are rather complicated, but maybe https://betamasaheft.eu/works/LIT2003Mazmur.xml can nevertheless be useful as an example. 
It would be great, if possible, if the precise Mazmura Krəstos ID could be identified (probably LIT2003Mazmur or LIT2002Mazmur)? For LIT2003Mazmur, sub-IDs have already been included in the record.

**Comment by Ralph-Lee-UK** [comment]
@thea-m and @DenisNosnitsin1970 this is very helpful. I see from the database that there is LIT5852Psalter which is a non-standard psalter, and non standard because it does not contain the standard five components. It would make sense to use this for the overall work, and then use the correct Mazmura Krestos for the first part, then followed by the other standard items, I think?
